insomnia.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Name=Insomnia
+Comment=Debug APIs like a human, not a robot
+Exec=/app/bin/insomnia %U
+Terminal=false
+Type=Application
+Icon=insomnia
+MimeType=x-scheme-handler/insomnia;
```

Comment:
Could use some keywords if you want to make the app easily discoverable

Reply:
Note sure in which context these are used, shall I just copy the ones upstream uses on their git repo?

```suggestion
MimeType=x-scheme-handler/insomnia;
Keywords=api,graphql,curl,rest-api,grpc,http-client,api-client,electron-app,api-design
```

Reply:
they will be used for search on Flathub & other software centers. Also on the gnome shell search menu (and probably other DE as well)

Reply:
Sounds like the desktop file and the AppStream have _some_ overlap, right?